fury/pick.py

```diff
@@ -73,3 +77,113 @@ def event_position(self, iren):
             using providing ShowManager's iren attribute.
         """
         return iren.GetEventPosition()
+
+    def pickable_on(self, actors):
+
+        if isinstance(actors, Sequence):
+            for a in actors:
+                a.PickableOn()
+        else:
+            actors.PickableOn()
+
+    def pickable_off(self, actors):
+
+        if isinstance(actors, Sequence):
+            for a in actors:
+                a.PickableOff()
+        else:
+            actors.PickableOff()
+
+
+class SelectionManager():
+
+    def __init__(self, select='faces'):
+        self.hsel = vtk.vtkHardwareSelector()
+        self.update_selection_type(select)
+
+    def update_selection_type(self, select):
+        self.selected_type = select
+        if select == 'faces' or select == 'edges':
+            self.hsel.SetFieldAssociation(
+                vtk.vtkDataObject.FIELD_ASSOCIATION_CELLS)
+        if select == 'points' or select == 'vertices':
+            self.hsel.SetFieldAssociation(
+                vtk.vtkDataObject.FIELD_ASSOCIATION_POINTS)
+        if select == 'actors':
+            self.hsel.SetActorPassOnly(True)
+
+    def pick(self, disp_xy, sc):
+        return self.select(disp_xy, sc, area=0)[0]
+
+    def select(self, disp_xy, sc, area=0):
+        info_plus = {}
+        self.hsel.SetRenderer(sc)
+        if isinstance(area, int):
+            picking_area = area, area
+        else:
+            picking_area = area
```

Comment:
if it is not a `int`, what are you expecting?  needs to be clarify in the docstring